index.js

```diff
@@ -0,0 +1,352 @@
+'use strict';
+
+const Scm = require('screwdriver-scm-base');
+const async = require('async');
+const hoek = require('hoek');
+
+process.on('unhandledRejection', console.dir);
```

Comment:
I've been pretty curious about this, too. I feel like this would cause some unexpected log output dependent entirely on whether or not this module is loaded. This could be really useful to do at the api level and leverage the api's logging system, instead.